<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>160</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 134, 28);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>296</y>
      <width>210</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>194</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>228</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>228</y>
      <width>103</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>194</y>
      <width>103</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PosNegButton">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>160</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(150, 149, 159);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumDisplay">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>20</y>
      <width>427</width>
      <height>95</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: white;
	border: 1px solid gray;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="EqualsButton">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>296</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 142, 108);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>262</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>121</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(90, 90, 90);
	color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SubtractButton">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>228</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 142, 108);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>262</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>262</y>
      <width>103</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>194</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MultiplyButton">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>194</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 142, 108);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButtton">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>262</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 142, 108);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DivideButton">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>160</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 142, 108);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>228</y>
      <width>102</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecimalButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>296</y>
      <width>103</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(169, 177, 208);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PercentButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>103</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(150, 149, 159);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: white;
	border: 1px solid gray;
	padding: 5px;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
